eng/mgmt/mgmtmetadata/compute_resource-manager.txt

```diff
@@ -3,13 +3,13 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=C:\Users\roip\dev\azure-sdk-for-net\sdk
+cmd.exe /c autorest.cmd https://github.com/rytruong/azure-rest-api-specs/blob/dev/rytruong/swagger_diskrp/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=C:\work\azure-sdk-for-net\sdk
```

Comment:
Should we update the information in this PR to the main branch of swagger repo since we already have the corresponding PR merged.